src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp

```diff
@@ -690,8 +690,7 @@ void PerformanceImplicitGemmBwdDataV4R1Xdlops::EuristicInit(const ConvolutionCon
     // final check
     if(!tmp.IsReallyValid(ctx))
     {
-        MIOPEN_LOG_E("All attempts failed");
-        assert(false);
+        MIOPEN_LOG_I("All attempts failed");
```

Comment:
This may produce unnecessary spam during `IsApplicable()` calls. Make it Info2 at least. Or, better, follow https://github.com/ROCmSoftwarePlatform/MIOpen/pull/595#pullrequestreview-531881874.

Reply:
#597 has been created to track the review comment